src/NHibernate.Test/CfgTest/Loquacious/LambdaConfigurationFixture.cs

```diff
@@ -21,50 +19,57 @@ public class LambdaConfigurationFixture
 		public void FullConfiguration()
 		{
 			var configure = new Configuration();
-			configure.SessionFactoryName("SomeName");
-			configure.Cache(c =>
-												{
-													c.UseMinimalPuts = true;
-													c.DefaultExpiration = 15;
-													c.RegionsPrefix = "xyz";
-													c.Provider<HashtableCacheProvider>();
-													c.QueryCacheFactory<StandardQueryCacheFactory>();
-												});
-			configure.CollectionTypeFactory<DefaultCollectionTypeFactory>();
-			configure.HqlQueryTranslator<ASTQueryTranslatorFactory>();
-			configure.LinqToHqlGeneratorsRegistry<DefaultLinqToHqlGeneratorsRegistry>();
-			configure.Proxy(p =>
-												{
-													p.Validation = false;
-													p.ProxyFactoryFactory<StaticProxyFactoryFactory>();
-												});
-			configure.Mappings(m=>
-								{
-									m.DefaultCatalog = "MyCatalog";
-									m.DefaultSchema = "MySche";
-								});
-			configure.DataBaseIntegration(db =>
-											{
-												db.Dialect<MsSql2000Dialect>();
-												db.KeywordsAutoImport = Hbm2DDLKeyWords.AutoQuote;
+			configure.ByCode(
+				x =>
+				{
+					x.SessionFactoryName("SomeName");
+					x.Cache(
+						c =>
+						{
+							c.UseMinimalPuts = true;
+							c.DefaultExpiration = 15;
+							c.RegionsPrefix = "xyz";
+							c.Provider<HashtableCacheProvider>();
+							c.QueryCacheFactory<StandardQueryCacheFactory>();
+						});
+					x.CollectionTypeFactory<DefaultCollectionTypeFactory>();
+					x.HqlQueryTranslator<ASTQueryTranslatorFactory>();
+					x.LinqToHqlGeneratorsRegistry<DefaultLinqToHqlGeneratorsRegistry>();
+					x.Proxy(
+						p =>
+						{
+							p.Validation = false;
+							p.ProxyFactoryFactory<StaticProxyFactoryFactory>();
+						});
+					x.Mappings(
+						m =>
+						{
+							m.DefaultCatalog = "MyCatalog";
+							m.DefaultSchema = "MySche";
+						});
+					x.DataBaseIntegration(
+						db =>
+						{
+							db.Dialect<MsSql2000Dialect>();
+							db.KeywordsAutoImport = Hbm2DDLKeyWords.AutoQuote;
 #if NETFX
-												db.Batcher<SqlClientBatchingBatcherFactory>();
+							db.Batcher<SqlClientBatchingBatcherFactory>();
```

Comment:
Interesting why CI throws here 
`CfgTest/Loquacious/LambdaConfigurationFixture.cs(56,19): error CS0246: The type or namespace name 'SqlClientBatchingBatcherFactory' could not be found`

Seems `if NETFX ` is ignored for some reasons